modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -323,4 +323,10 @@
             <class name="org.wso2.am.integration.tests.login.EmailUserNameLoginTestCase"/>
         </classes>
     </test>
+    <test name="apim-integration-tests-workflow" preserve-order="true" parallel="false" group-by-instances="true">
+        <classes>
+            <class name="org.wso2.am.integration.tests.other.AdvancedConfigDeploymentConfig"/>
```

Comment:
I don't think we need to add AdvancedConfigDeploymentConfig test. APIs are not invoked from your test.